docs/tutorials/quick-start.md

```diff
@@ -142,4 +142,4 @@ The output should now be:
 
 Well done, you've created your first Concerto model and used the CLI to validate data against the model! This is just the start...
 
-You may want to continue to read about [static code generation](./category/code-generation), or using the [JavaScript runtime API](/docs/reference/api/api-js-validation) to introspect models at runtime.
\ No newline at end of file
+You may want to continue to read about using the [JavaScript runtime API](/docs/reference/api/api-js-validation) to introspect models at runtime.
```

Comment:
The code generation page exists, please update the path accordingly https://concerto.accordproject.org/docs/category/code-generation